js/main.js

```diff
@@ -1,5 +1,13 @@
-import { generateOffers } from './data.js';
+import { generateOffers } from './offer.js';
+import { renderOffers } from './templete.js';
 
 const NUMBERS_OF_OFFERS = 10;
 
-generateOffers(NUMBERS_OF_OFFERS);
+const offers = generateOffers(NUMBERS_OF_OFFERS);
+
+const firstOffer = offers[0];
```

Comment:
```suggestion
const [firstOffer] = offers;
```

Почитай про реструктуризацию массивов